src/main/resources/templates/start.html

```diff
@@ -15,12 +15,12 @@
     <!-- Navigation Bar-->
     <div th:replace="fragments/reusable-components :: navbar"></div>
     <!-- Collection Section-->
-    <div class="container main-div">
+    <div class="container pt-4 pb-4">
         <div class="row row-cols-auto">
             <h2>Collections</h2>
             <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addCollectionModal">Add Collection</button>
         </div>
-        <div class="row row-cols-auto">
+        <div class="row row-cols-auto pt-1">
```

Comment:
The button in the heading for this div had zero padding, which made it look weird. Added padding-top (pt) to the row below it in order to get the spacing we needed between the elements.

Citation: https://getbootstrap.com/docs/4.0/utilities/spacing/